plugin/src/main/kotlin/org/jetbrains/research/pynose/plugin/inspections/TestRunnerGetter.kt

```diff
@@ -0,0 +1,25 @@
+package org.jetbrains.research.pynose.plugin.inspections
+
+import com.intellij.openapi.module.Module
+import com.intellij.openapi.module.ModuleUtilCore
+import com.intellij.psi.PsiFile
+import com.jetbrains.python.testing.TestRunnerService
+
+object TestRunnerGetter {
```

Comment:
And I don't like this name, word _getter_ has its own meaning in OOP. Let's use smth like `TestRunnerServiceFacade`

Reply:
Moreover, why do you even need this class?) There is almost no logic here, why not just use Kotlin Extensions, if you want to avoid copy-paste?

Reply:
I don't fully understand what you meant by that. I created a class because I thought it's logic differs from the one of  abstract inspection class. however, I agree with the fact that there is almost no logic in it. do you want me to add this logic to abstract inspection as extensions or smth else?..